go.mod726238372.tmp

```diff
@@ -0,0 +1,17 @@
+module github.com/CESARBR/knot-cloud-storage
```

Comment:
Should we add this file?

Reply:
Nope, generated `.tmp` files shouldn't be committed :stuck_out_tongue:

Reply:
Sorry :smile: Already remove it!